Firestore/core/src/remote/remote_store.cc

```diff
@@ -59,13 +59,15 @@ using util::Status;
 constexpr int kMaxPendingWrites = 10;
 
 RemoteStore::RemoteStore(
-    LocalStore* local_store,
+    local::LocalStore* local_store,
```

Comment:
Nit: qualifying `LocalStore` with its namespace shouldn't be necessary given the `using` directive above.

Reply:
Fixed